lib/reek/ast/sexp_extensions/attribute_assignments.rb

```diff
@@ -0,0 +1,10 @@
+module Reek
+  module AST
+    module SexpExtensions
+      # Utility methods for :attrasgn nodes.
+      module AttrasgnNode
+        def args() children[2] end
```

Comment:
Can we unwind this and all the other single-line method definitions (e.g. in `AST::SexpExtensions::BlockNode`)? This throws my mental parser a tiny bit off every single time (‘why parentheses? why no semicolon?’) and I don’t think we’re going to run off of file lines anytime soon. ;)